gprofiler/metadata/application_identifiers.py

```diff
@@ -287,6 +293,35 @@ def get_app_id(self, process: Process) -> Optional[str]:
         return None
 
 
+class _JavaSparkApplicationIdentifier(_ApplicationIdentifier):
+    _JAVA_SPARK_EXECUTOR_ARG = "org.apache.spark.executor.CoarseGrainedExecutorBackend"
+    _SPARK_PROPS_FILE = os.path.join("__spark_conf__", "__spark_conf__.properties")
+    _APP_NAME_NOT_FOUND = "SPARK_APP_NAME_NOT_FOUND"
+    _APP_NAME_KEY = "spark.app.name"
+
+    @staticmethod
+    def _is_java_spark_executor(process: Process) -> bool:
+        args = process.cmdline()
+        if not _is_java_bin(args[0]):
```

Comment:
Is `_JAVA_SPARK_EXECUTOR_ARG` not enough? Why check for `java` etc as well? And why `jsvc` etc, can it happen? I thought jsvc was meant for other types of services.